analyzers/src/ITs.JsonParser/Json/SarifReport.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace ITs.JsonParser.Json;
+
+// Format of JSONs in "output/"
+internal class SarifReport
+{
+    public SarifRun[] Runs { get; set; }
+
+    public SarifIssue[] AllIssues() => Runs.SelectMany(x => x.Results).ToArray();
+}
+
+internal class SarifRun
+{
+    public SarifIssue[] Results { get; set; }
+}
+
+internal class SarifIssue
+{
+    public string RuleId { get; set; }
+    public string Message { get; set; }
+    public SarifLocation[] Locations { get; set; }
+
+    public Location Location => Locations is { Length: > 0 } ? Locations[0].ResultFile : null;
+
+    public object Order() =>
+        Location is null
+            ? default
```

Comment:
Coding style (should use null where we know it's reference)
```suggestion
            ? null
```